packages/iris-grid/src/IrisGridPartitionSelector.tsx

```diff
@@ -322,7 +322,7 @@ class IrisGridPartitionSelector extends Component<
                 ? null
                 : (partitionConfig.partitions[index] as ItemKey)
             }
-            placeholder={'Select an option' as string}
+            placeholder={'Select a key' as string}
```

Comment:
Well this is odd, I don't know why it was like this (casting a string to a `string`). The casting is unnecessary.
```suggestion
            placeholder="Select a key"
```